G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google.Tests/Commands/OcrGoogle.Find/ShouldReturnRectanglePosition_WhenTextIsFound.G1ANT

```diff
@@ -9,7 +9,7 @@ ocrgoogle.login ♥credential⟦ocrgoogle:jsoncredential⟧
 ocrgoogle.find ‴alice‴ area ♥screen result ♥rectangleResult
 ie.close
 
-test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32)⊃
-test ⊂♥rectangleResult⟦top⟧.GetType() == typeof(Int32)⊃
-test ⊂♥rectangleResult⟦right⟧.GetType() == typeof(Int32)⊃
-test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32)⊃
+test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32) && ♥rectangleResult⟦left⟧ > 0⊃
+test ⊂♥rectangleResult⟦top⟧.GetType() == typeof(Int32) && ♥rectangleResult⟦top⟧ > 0⊃
+test ⊂♥rectangleResult⟦right⟧.GetType() == typeof(Int32) && ♥rectangleResult⟦right⟧ > 0⊃
+test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32) && ♥rectangleResult⟦left⟧ > 0⊃
```

Comment:
No need to check the type, since comparison with another number will be enough + it's not a responsibility of this test to check the types of `rectangle` fields